README.md

```diff
@@ -1332,6 +1333,26 @@ See the [example-cron.yml](./.github/workflows/example-cron.yml) workflow.
 
 [![cron example](https://github.com/cypress-io/github-action/workflows/example-cron/badge.svg?branch=master)](.github/workflows/example-cron.yml)
 
+### Publish tests summary
```

Comment:
Change to "Suppress test summary"
due to decision to keep the default that summaries are published.